basis/http/http.factor

```diff
@@ -152,6 +159,7 @@ redirects ;
         <url>
             H{ } clone >>query
         >>url
+        <url> >>proxy-url
```

Comment:
You want f in ```proxy-url``` but an empty url with an empty query in ```url``` ? Or both to f ?

Also, this was a way to have a proxy-url fill in missing parts, so that for example you could set the credentials globally and the host/port for each request (or any combination). And it works the same for entirely setting the proxy globally. Is this feature useful ?